app/retail/templates/shared/footer_scripts.html

```diff
@@ -22,6 +22,11 @@
 </script>
 <script type="text/javascript" src="{% url 'javascript-catalog' %}"></script>
 <script src="{% static "v2/js/jquery.js" %}"></script>
+{% if env == 'prod' %}
+  <script src="https://cdn.jsdelivr.net/npm/vue"></script>
```

Comment:
can we host internally?

Reply:
Yes we can but in this way we use their cdn and not need to manage versions, what is the vantage of hosting that internally?

Reply:
is vue.js the direction we want to go globally? @mbeacom @SaptakS @thelostone-mc

Reply:
We decide to make a test with notifications so we can see how it works and then decide/discuss.

Reply:
> Yes we can but in this way we use their cdn and not need to manage versions, what is the vantage of hosting that internally?

- you can run on your local if you dont have internet
- 1 less DNS lookup
- we control the caching / distribution of the file..